Makefile

```
@@ -0,0 +1,61 @@
+
+IMAGE_NAME = onitama-web
+CONTAINER_NAME = onitama-web-container
+PORT = 3000
+
+help: ## Show this help message
+	@echo 'Usage: make [target]'
+	@echo ''
+	@echo 'Available targets:'
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "  %-15s %s\n", $$1, $$2}'
+
+dev: ## Run development environment locally with Docker
```

checkmake (0.2.2)

[warning] Target body for "dev" exceeds allowed length of 5 (13).

(maxbodylength)